packages/react-native-win32/.flowconfig

```diff
@@ -72,11 +75,6 @@
 ; Ignore unexpected extra "@providesModule"
 .*/node_modules/.*/node_modules/fbjs/.*
 
-; Ignore duplicate module providers
-; For RN Apps installed via npm, "Libraries" folder is inside
-; "node_modules/react-native" but in the source repo it is in the root
-.*/Libraries/react-native/React.js
```

Comment:
>native [](start = 19, length = 6)

the react-native dir got nuked, but React.js didn't exist in there anyway? Cleaned this up